### DISCUSSION TITLE
Settings -> Automations: imperferction of an "actions" column's label

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#21619] [Configuration]
Settings -> Automations:
![image](https://github.com/user-attachments/assets/9b648f91-638b-4d27-a150-f8a06dc198d9)

All columns are starting with capitals - except "actions".
Ofc this is not an entity's property - but suggest to rename it to "Actions" (looks nicer).

Same for Scripts, Blueprints, ...

**Comment by karwosts** [comment]
Just a tip, generally when you see a word in the UI that looks like it should be capitalized, but is not, that's probably because it's not being translated. So in other languages this word would not be translated (and so this is probably more of a minor localization bug).

**Reply by ildar170975** [reply]
Shall I register a bug then?
Is it possible to convert a discussion to an issue?

**Reply by karwosts** [reply]
I can move it, but I was fixing it anyway, should be ready soon.

**Reply by karwosts** [reply]
Well guess that's more of a duplication than a move 😅 . But it should be fixed shortly.

**Reply by ildar170975** [reply]
OK, then will check it then))
